docs/DeveloperGuide.md

```diff
@@ -378,234 +375,361 @@ Use case ends.
 
 
 * 2b. SellSavvy detects that the new customer has tags with similar names.
-  * 2b1. SellSavvy gives a warning that there are similar tags in the new customer.
+  * 2b1. SellSavvy gives a warning that there are similar tags on the new customer.
   
   Use case resumes from step 3.
 
 **Use case 3: Delete Customer and All Orders Related to The Customer**
 
-* **System**: SellSavvy
-* **Actor**: User
 * **Use Case**: UC03 - Delete Customer and All Orders related to the customer
 * **Preconditions**: There are customers displayed in the customer list.
 * **Guarantees**:
     * Customer and all their orders will be deleted if input parameters are valid.
 
 **MSS**
 
-1. User wants to delete a customer.
-2. User finds the customer index from the list.
-3. User deletes the customer by their index.
-4. SellSavvy updates the displayed list of customers and indicates that delete is successful.
+1. User finds the customer index from the list.
+2. User deletes the customer by their index.
+3. SellSavvy updates the displayed list of customers.
```

Comment:
Since we updated the UG that deletecustomer clears the order list too, do we need to update the DG here?